src/frontend/templates/components/Navbar.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import '../../styles/components/navbar.css';
+import { Link } from 'react-router-dom';
+
+export const Navbar = () => {
+
+    const [navlinksClass, setNavlinksClass] = useState('nav-links');
+    const [navSearchClass, setSearchClass] = useState('nav-g-search');
+
+    const toggleNavBar = (e) => {
+        navlinksClass === 'nav-links'
+          ? setNavlinksClass('nav-links-responsive')
+          : setNavlinksClass('nav-links');
+        navSearchClass === 'nav-g-search responsive'
+          ? setSearchClass('nav-g-search')
+          : setSearchClass('nav-g-search responsive');
+    };
```

Comment:
I think this snippet can be simplified in the following way.

<element className={"common-class" + (stateVariable ? "conditional-class" : "")}

Reply:
hi tejas , thanks for the suggestion , will implement this in future iteration !